config/others/linux-tutorial.yml

```diff
@@ -1,14 +1,7 @@
 site_name: Linux tutorial
-docs_dir: docs/linux-tutorial
+docs_dir: docs/linux
```

Comment:
We should keep `linux-tutorial` as directory name here, to avoid breaking the existing URL https://hpcugent.github.io/vsc_user_docs/linux-tutorial/ ?